packages/core/test/util/renderer.test.ts

```diff
@@ -460,3 +524,35 @@ test('mapDispatchToArrayControlProps should remove items from array', t => {
   t.is(store.getState().jsonforms.core.data.length, 1);
   t.is(store.getState().jsonforms.core.data[0], 'quux');
 });
+
+test('mapStateToLayoutProps - visible via state with path from ownProps ', t => {
```

Comment:
Maybe we should also a test case that expects visible to be false

Reply:
yes we should do so

Reply:
Added a commit providing the test case